types/klaw-sync/index.d.ts

```diff
@@ -1,40 +1,62 @@
-// Type definitions for klaw-sync 1.1
+// Type definitions for klaw-sync 3.0
```

Comment:
Does this need to go up to `3.0`? Can it stay on `2.0` to be in sync with the version of the dependency project?